python/examples/core/harness.py

```diff
@@ -74,6 +76,81 @@ def to_data_frame(self) -> pandas.DataFrame:
     """Return a data frame containing the aggregated data."""
     return self.data
 
+  def plot(self, path: os.path, data_key: str, data_label: str):
+    """Plot the measurements."""
+    config_key = "runtime_problem_sizes_dict"
+    keys_to_iter = [k for k in self.config_keys if k is not config_key]
+    config_to_iter = self.data[keys_to_iter].drop_duplicates()
+    for _, row in config_to_iter.iterrows():
+      # Compute the plot file name by filtering non alpha numberic characters.
```

Comment:
seems like this should be extracted out in a separate helper func

Reply:
done